trace-shield/helm/trace-shield/values.yaml.tpl

```diff
@@ -0,0 +1,69 @@
+{{ $hydraPostgresPass := dedupe . "trace-shield.postgresHydra.password" (randAlphaNum 32) }}
+postgresHydra:
```

Comment:
couldn't we have all three of these share the same db?

Reply:
(you'd need to have the postgres operator create multiple dbs w/in the same one of course)